system/forms/widgets/iframewidget.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<form>
```

Comment:
We should use the newish `i18nBaseUri` attribute on the `form` element here for all the labelling. So `<form i18nBaseUri="widgets.iframewidget:">`. You can then remove all the label attributes from the fields and use the following convention in the `.properties` file:

    field.url.title=URL
    field.url.placeholder=e.g. http://myiframesite.com/iframe.html
    field.url.help=Some help text about URL

etc.